public/assets/i18n/app/en.json

```diff
@@ -29,6 +29,8 @@
   "default-language": "Default language",
   "source-lang-not-found": "Untranslated. Default: $1",
   "source-to-target": "to",
+  "confirmation-to-switch-target-lang": "You are trying to change the translation language but you have unsaved translations. Changing the language will result in loss of any added translations. Please upload the translations to Commons or download the translated file before proceeding. Do you wish to continue?",
+  "confirm-change-target-lang": "Change language",
```

Comment:
Why do we duplicate the translations in public/assets/i18n?

Reply:
Could you use a symlink instead?

Reply:
No, becuase one of the copied dirs is `node_modules/jquery.uls/i18n` which doesn't exist in the repo.